tests/clang-check/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM fedora:latest
 
-RUN yum install -y make clang-tools-extra clang python3-pip 'dnf-command(builddep)' && \
+RUN dnf install protobuf-c -y && yum install -y make clang-tools-extra clang python3-pip 'dnf-command(builddep)' && \
```

Comment:
Why not `yum install -y protobuf-c make clang-tools-extra clang ...'`?

Reply:
@rst0git Sure but i think `protobuf-c` is wrong dependency.

Reply:
Changing it to `protobuf-c-devel`

Reply:
fixed :)